api/data_explorer/__main__.py

```diff
@@ -272,6 +272,20 @@ def _get_es_facets():
     return facets
 
 
+def _get_table_names():
+    """Gets an alphabetically ordered list of table names from bigquery.json.
+    Table names are fully qualified: <project id>:<dataset id>:<table name>
```

Comment:
Standard BQ syntax is to use periods. Change to:

<project id>.<dataset id>.<table name>

Reply:
Done